docs/content/about-iota/iota-architecture/consensus.mdx

```diff
@@ -5,15 +5,49 @@ description: Overview of the IOTA consensus.
 tags: [core-protocol]
 ---
 
-The basic purpose of consensus in blockchains is to agree on a consistent order and ensure the availability of transactions.
+The primary goal of consensus in distributed systems is to establish a consistent order of transactions and ensure their availability. In IOTA, consensus is achieved through a combination of Delegated Proof-of-Stake (dPoS) and the Mysticeti Byzantine Fault Tolerant (BFT) consensus protocol.
 
-On IOTA, consensus has a simple API: validators submit different user transactions to consensus concurrently, and the consensus outputs a consistent stream of transactions across all well-behaving validators.
+IOTA's consensus mechanism ensures:
 
-Byzantine-fault tolerant (BFT) consensus protocols are a rich area of research. The next-generation consensus engine in IOTA is based on the **Mysticeti** protocol.
-The protocol optimizes for both low latency and high throughput because:
-- It allows multiple validators to propose blocks in parallel, utilizing the full bandwidth of the network and providing censorship resistance. These are features of the DAG-based consensus protocols.
-- It takes only three rounds of messages to commit blocks from the DAGs, same as pBFT and matches the theoretical minimum.
-- The commit rule allows voting and certifying leaders on blocks in parallel, further reducing the median and tail latencies.
-- The commit rule also tolerates unavailable leaders without significantly increasing the commit latencies.
+- Low latency: Transactions are confirmed quickly due to efficient leader election and consensus mechanisms.
+- High throughput: The protocol can process a large number of transactions per second, ensuring scalability.
+- Fast finality: Transactions reach finality in a short time, reducing the risk of rollbacks.
+- Energy efficiency: Unlike Proof-of-Work (PoW), Mysticeti's consensus mechanism requires minimal computational resources.
+- Security & decentralization: The protocol resists common attacks by relying on an honest majority of validators while maintaining a decentralized structure.
```

Comment:
Please use bold for the key points

Reply:
Oh, sorry about that, I missed it somehow.